harmony/harmonydb/sql/20240930-pdp.sql

```diff
@@ -85,6 +86,11 @@ CREATE TABLE pdp_proof_sets (
     create_message_hash TEXT NOT NULL,
     service TEXT NOT NULL REFERENCES pdp_services(service_label) ON DELETE RESTRICT
 );
+ALTER TABLE pdp_proof_sets 
+ADD COLUMN proofset_listener TEXT NOT NULL, -- the listener is assumed to be a PDP Service with proving window informational methods
+ADD COLUMN proving_period BIGINT NOT NULL, -- the proving period for this proofset
+ADD COLUMN challenge_window BIGINT NOT NULL, -- the challenge window for this proofset
+ADD COLUMN prove_at_epoch BIGINT; -- the epoch at which the next challenge window starts and proofs can be submitted
```

Comment:
I'd just add this to the main CREATE TABLE above

Reply:
Also can those values just be pulled from the contract getters when scheduling? Better to avoid duplicated state in the database when feasible